contracts/mocks/Cover/CoverMockStakingPool.sol

```diff
@@ -43,13 +41,13 @@ contract CoverMockStakingPool is IStakingPool, ERC721 {
     address /* _coverContract */,
     ITokenController /* _tokenController */,
     address _memberRoles
-  ) ERC721("Nexus Mutual Staking Pool", "NMSPT")
+  ) ERC721Mock("Nexus Mutual Staking Pool", "NMSPT")
   {
     memberRoles = _memberRoles;
   }
 
-  function name() public view override returns (string memory) {
-    return string(abi.encodePacked(super.name(), " ", Strings.toString(poolId)));
+  function nameWithPoolId() public view returns (string memory) {
```

Comment:
I think the `name()` function in this mock was only present because it wasn't implemented in the inherited contract at the time so it wasn't deployable. We probably need to delete this function from the mock, but search for usages first to make sure we don't actually use it.

Reply:
Ok. Looks like it's not being called.